File: src/main/java/io/github/oleksivio/telegram/bot/api/model/method/location/EditOtherMessageLiveLocation.java
package io.github.oleksivio.telegram.bot.api.model.method.location;

import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker;
import io.github.oleksivio.telegram.bot.core.model.method.location.EditMessageLiveLocation;
import io.github.oleksivio.telegram.bot.core.model.responses.BooleanResponse;
import io.github.oleksivio.telegram.bot.core.model.responses.CommonResponse;

public class EditOtherMessageLiveLocation extends EditMessageLiveLocation<Boolean> {

    public EditOtherMessageLiveLocation(ActionNetworker actionNetworker) {
        super(actionNetworker);
    }

    @Override
    protected Class<? extends CommonResponse<Boolean>> getResultWrapperClass() {
        return BooleanResponse.class;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/method/location/EditOtherMessageLiveLocation.kt
package io.github.oleksivio.telegram.bot.api.model.method.location

import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker
import io.github.oleksivio.telegram.bot.core.model.BooleanResponse
import io.github.oleksivio.telegram.bot.core.model.method.location.EditMessageLiveLocation

class EditOtherMessageLiveLocation(actionNetworker: ActionNetworker) : EditMessageLiveLocation<Boolean>(actionNetworker) {
    override val resultWrapperClass = BooleanResponse::class
}
